pyproject.toml

```diff
@@ -57,10 +56,12 @@ line-length = 120
 log_level = "INFO"
 
 [tool.black]
-exclude = [
-    "tests/_data/",
-    "tests/_data_clean/",
-    "tests/_data_duplicated_deps/",
-    "tests/_data_ignore/",
-    "tests/_invalid_data/",
-]
+exclude = '''
+/(
```

Comment:
whats the difference between this and the old format? it was more readable/pythonic, was it broken?